ui/aura/mus/window_tree_client.cc

```diff
@@ -2269,4 +2276,4 @@ void WindowTreeClient::OnCompositingShuttingDown(ui::Compositor* compositor) {
   compositor->RemoveObserver(this);
 }
 
-}  // namespace aura
\ No newline at end of file
+}  // namespace aura
```

Comment:
Is it OK if I handle it during the rebase step? Our other patches introduced it and git always complains if I change window_tree_client.cc somehow.